.github/ISSUE_TEMPLATE/pre-work-template-devops-security.md

```diff
@@ -0,0 +1,74 @@
+---
+name: Pre-work Template - DevOps-Security
+about: Newly onboarded team members should start by making this issue for themselves
+title: 'Pre-work Checklist: DevOps-Security-Member: [replace brackets with your name]'
+labels: 'Complexity: Prework, Feature: Onboarding/Contributing.md, role missing, size:
+  1pt'
+assignees: ''
+
+---
+
+### Prerequisite
+We are looking forward to having you on our team. Please ensure you have prior experience with the HfLA website team before contributing to our repository.
+### Overview
+As a new member on the HfLA devops-security team, fill in the following fields as you complete each onboarding item.
+
+### Special Notes
+1. Keep this issue open until you've completed all steps, including learning to provide updates for longer tasks.
+2. Normally, handle one issue at a time, but this one is an exception as it teaches how to manage multiple tasks.
+3. Work on action items sequentially, but proceed if possible. For example, set up your dev environment without waiting for the weekly meeting.
+
+### Action Items
+
+- [ ] Before starting to work on the below instructions, make sure to join the #ops Slack Channel. And are a member of `devops-security` repository.
+- [ ] Self-assign this issue (gear in right side panel).
+- [ ] Add this issue to the Project Board under the Projects section (gear in right side panel).
+- [ ] Attend weekly team meeting, Wednesdays 6-8pm PST.
+  - [ ] Note: There are no meetings on the 1st-7th of every month.
+- [ ] Complete the steps in [Creating a personal AWS account](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#creating-a-personal-aws-account) and [Login as root user & setup MFA](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#login-as-root-user-&-setup-mfa).
+- [ ] Read and follow the instructions in [Setting up IAM and AWS CLI](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#setting-up-iam-and-aws-cli) for :-
+      - [ ] [Creating an IAM User](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#create-an-iam-group)
+      - [ ] [Creating an IAM Group] (https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#create-an-iam-group)
+      - [ ] [Attaching IAM user to IAM Group] (https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#attach-iam-user-to-iam-group)
+      - [ ] [Providing `AdministratorAccess` policy to IAM group] (https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#attach-administratoraccess-policy-to-iam-group)
+      - [ ] Log in as the newly created user instead of continuing to log in as the root user (it is not recommended to login with root access).
+      - [ ] [Generating user access keys] (https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#generating-access-keys-for-aws-cli)
```

Comment:
```suggestion
- [ ] Read and follow the instructions in [Setting up IAM and AWS CLI](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#setting-up-iam-and-aws-cli) for:
  - [ ] [Creating an IAM User](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#create-an-iam-group)
  - [ ] [Creating an IAM Group](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#create-an-iam-group)
  - [ ] [Attaching IAM user to IAM Group](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#attach-iam-user-to-iam-group)
  - [ ] [Providing `AdministratorAccess` policy to IAM group](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#attach-administratoraccess-policy-to-iam-group)
  - [ ] Log in as the newly created user instead of continuing to log in as the root user (it is not recommended to login with root access).
  - [ ] [Generating user access keys](https://github.com/hackforla/devops-security/blob/main/CONTRIBUTING.md#generating-access-keys-for-aws-cli)
```